src/view/com/auth/SplashScreen.web.tsx

```diff
@@ -98,22 +99,23 @@ export const SplashScreen = ({
 
 function Footer({styles}: {styles: ReturnType<typeof useStyles>}) {
   const pal = usePalette('default')
+  const {_} = useLingui()
 
   return (
     <View style={[styles.footer, pal.view, pal.border]}>
       <TextLink
         href="https://blueskyweb.xyz"
-        text="Business"
+        text={_(msg`Business`)}
         style={[styles.footerLink, pal.link]}
       />
       <TextLink
         href="https://blueskyweb.xyz/blog"
-        text="Blog"
+        text={_(msg`Blog`)}
```

Comment:
We probably want to revert this as wella

Reply:
But in HomeLoggedOutCTA.tsx, it was already localized.